distribution/tools/server-cli/src/main/java/org/elasticsearch/server/cli/SystemJvmOptions.java

```diff
@@ -155,7 +155,7 @@ private static String findLibraryPath(Map<String, String> sysprops) {
         }
         String archname = sysprops.get("os.arch");
         String arch;
-        if (archname.equals("amd64")) {
+        if (archname.equals("amd64") || archname.equals("x86_64")) {
```

Comment:
I think that this should be fine, and cover the archs we run on. No need for archs scuh as, say, "i386", "x86", "ppc64", "ppc64le", "aarch64", "riscv64".